MeNext/Layout/SearchView.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Xamarin.Forms;
+
+
+namespace MeNext
+{
+	public class SearchView : ContentPage
+	{
+		private SearchBar searchBar;
+		private Label resultsLabel;
+		private SongListView songList;
+
+		public SearchView()
+		{
+			// TODO: clean up this testing code
+			this.Title = "Search";
+			NavigationPage.SetHasNavigationBar(this, false);
```

Comment:
By default these types of pages do have a Navigation Bar thing at the top, but only the overall page actually has/needs one, this line just makes it so there isn't a weird empty line at the top of the page.